cookbooks/convection-box/tutorial-onset-of-convection/model_input/.ipynb_checkpoints/convection-box2-old-checkpoint.prm

```diff
@@ -0,0 +1,161 @@
+# A description of convection in a 2d box. See the manual for more information.
+
+
+# At the top, we define the number of space dimensions we would like to
```

Comment:
This and many other files are in a folder ``.ipynb_checkpoints/``. I think this temporary folder should not be part of your PR.

Reply:
Oh drats, you did find more. Do you mind if i clean this up when i get rid of the "old" files?